src/neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -41,6 +41,8 @@
             pip install tensorflow"""
         raise ModuleNotFoundError(msg) from exc
 
+from neptune.common.warnings import NeptuneWarning
```

Comment:
Please do not use internal code in integrations 🙏 . You can probably define some integration-specific warning.

Reply:
@Raalsky - any specific reason? Should I revert to using `RuntimeWarning`?

Reply:
We can change it anytime cause it's not documented anywhere :wink:. And actually, we already moved the whole `neptune.common` in one of the major updates. Maybe you could define a new warning class right here? Doesn't look to be much complicated.

Reply:
Yeah.. I saw the PR removing `neptune.common` after pushing this change 🙈 
I'll revert to `RuntimeWarning` (unless you've a better category in mind). This is not big/important enough yet to warrant it's own warning class.